src/stories/Library/pause-reservation/PauseReservation.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { PauseReservation } from "./PauseReservation";
+
+export default {
+  title: "Library / Pause reservation",
+  component: PauseReservation,
+  decorators: [withDesign],
```

Comment:
Since you are using withDesign I think you should specify an url to the Figma design ;)